masterlist.yaml

```diff
@@ -198,7 +203,7 @@ globals:
 # SKSE64 plugins requirement
   - <<: *requireSKSE64
     subs: [ 'a SKSE Plugin' ]
-    condition: 'file("SKSE/Plugins/([^\.]+\.dll)") and not file("../skse64_loader.exe")'
+    condition: 'file("SKSE/Plugins/([^\.]+\.dll)") and not file("../skse64_loader.exe") and not file("../sksevr_loader.exe")'
```

Comment:
Are the VR checks even required?

Reply:
Yes, if it's not there then SKSEVR users will see messages about not having SKSE installed.